repos/system_upgrade/common/models/ipuconfig.py

```diff
@@ -96,3 +106,6 @@ class IPUConfig(Model):
 
     The list contains only upgrade paths for the `flavour` of the source system.
     """
+
+    distro = fields.Model(Distro)
+    "Release IDs of the source and target system."
```

Comment:
```suggestion
    """Release IDs of the source and target system."""
```

we require all docstring to use triple-quotes

Reply:
Yeah, I know, not sure why I put it like this